app/knight.py

```diff
@@ -0,0 +1,30 @@
+class Knight:
+    def __init__(self, knight: dict) -> None:
+        self.name = knight["name"]
+        self.power = knight["power"]
+        self.hp = knight["hp"]
+        self.armour = knight["armour"]
+        self.weapon = knight["weapon"]
+        self.potion = knight["potion"]
```

Comment:
that's pretty bad approach, since you should create knight not from dict, you can use some manager that translates dict to parameters, but not unpack dict like that.